.editorconfig

```diff
@@ -23,21 +23,21 @@ spelling_exclusion_path = SpellingExclusions.dic
 # Code files
 [*.{cs,csx,vb,vbx}]
 indent_size = 4
-insert_final_newline = true
-trim_trailing_whitespace = true
-charset = utf-8
 
 # XML project files
 [*.{csproj,vbproj,vcxproj,vcxproj.filters,proj,projitems,shproj}]
 indent_size = 2
+insert_final_newline = false
```

Comment:
Why this change?

Reply:
@cschuchardt88,
 maybe it is better to no change anything now because my idea was to get the files and verify its integrity with a hash function in order to ensure no changes were done.

Reply:
@shargon, what is the best way to verify any change in any file?

Reply:
> Why this change?

Its a sample change, these files shouldn't have throughs at the end. But it can be removed.

> @shargon, what is the best way to verify any change in any file?

Tests? use a compare file tool?